facial-age-estimator/src/facial-age-estimator-output.js

```diff
@@ -0,0 +1,8 @@
+/* globals tf */
+
+const postprocess = function (inferenceResults) {
+  return inferenceResults.array()
```

Comment:
https://github.com/IBM/MAX-Facial-Age-Estimator returns
```
{
    "status": "ok",
    "predictions": [
        {
            "age_estimation": 47,
            "detection_box": [
                0.3079667063020214,
                0.147,
                0.6813317479191439,
                0.562
            ]
        }
    ]
}
```

would be possible to make this return something similar? for example,
```
[ {
    ageEstimation: 47,
    detectionBox: [ 0.3079667063020214, 0.147, 0.6813317479191439, 0.562 ]
  }, {
    ageEstimation: 41,
    detectionBox: [ 0.706, 0.147, 0.747919, 0.562 ]
} ]
```

Reply:
In this MAX model, we use another library wrapping a DL model to detect faces and crop them before sending them to the model to run the prediction. 
https://github.com/IBM/MAX-Facial-Age-Estimator/blob/104bde46d900d646689b64cd46f1d02e42204a52/core/model.py#L74
I can use a similar library and add that to the preprocessing step.

Reply:
oh ok. so fae does do any face detection localization.
alright for now we can hold off on that until we get rest of this working on in good shape?